Marvel/Characters/Thor/Thor (CBRO).cbl

```diff
@@ -0,0 +1,1249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ReadingList xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
While I don't think it's going to be a problem, I don't want to take the chance by having this line broken up. It could trip up the parsers.

Reply:
I fixed this issue as well as a number of other small issues by running it through my script.